elo_boat.py

```diff
@@ -1,18 +1,15 @@
 import discord
-from discord.ext import commands, tasks
+from discord.ext import commands
 import requests
 import os
 from itertools import combinations
 import sqlite3
-from dateutil.parser import parse
 import logging
 import json
 import trueskill
 import sys
-import math
 import random
 import csv
-import datetime

```

Comment:
Did we import useless stuff ? 
It's unlikely because i only add libraries that were needed to deal with modulenotfound error

=> maybe code was removed but even tho i recommend letting them here so that we can save time if we need them again

Reply:
Will revert that change.